### PR TITLE
feat: update to `core` version of `d2l-icons` and `d2l-colors`

### PR DESCRIPTION
**Pull request by devpow112**
Both of these are drop in replacements. The packages just directly import the `core` file. Didn't remove `d2l-typography` as it uses some of the style shims. This also switched to use GitHub Action, since we don't want to support Travis anymore, and `semantic-release` for ease of release.

### CODE REVIEW

File: test/d2l-save-status.js [hunk 1]
```diff
@@ -19,15 +19,4 @@ suite('<d2l-save-status>', function() {
 		});
 	});
 
-	suite ('Ally Test', function() {
```
Comment (devpow112) [inline test/d2l-save-status.js:22]: This didn't seem useful and was causing issues trying to get CI running so I removed it.

**Comment by devpow112** [comment]
@dlockhart can you kake sure that `brightspace-bot` is added as admin for this repo. Looks like there's no branch protection but might as well add it.

File: package.json [hunk 2]
```diff
@@ -12,13 +12,13 @@
   },
   "homepage": "https://github.com/Brightspace/save-status",
   "name": "d2l-save-status",
-  "version": "3.9.0",
+  "version": "3.10.0",
```
Comment (devpow112) [inline package.json:15]: Had to manually update this to be inline with latest tag.

**Comment by devpow112** [comment]
@dlockhart sorry about the false start on this one. It's good to review now. I'll be better about this in the future. Didn't think I was gonna have to do some much CI work.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+name: Release
+on:
+  push:
+    branches:
+      - main
```
Comment (dlockhart) [inline .github/workflows/release.yml:5]: Do you want me to rename `master` -> `main`? Otherwise this will need to change.

File: .travis.yml [hunk 4]
```diff
@@ -1,26 +0,0 @@
-language: node_js
```
Comment (dlockhart) [inline .travis.yml:1]: 🎉

File: d2l-save-status.js [hunk 5]
```diff
@@ -54,11 +53,11 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-save-status">
 			}
 		</style>
 		<div>
-			<d2l-icon icon="d2l-tier1:check" class="check-icon"></d2l-icon>
+			<d2l-icon icon="tier1:check" class="check-icon"></d2l-icon>
```
Comment (dlockhart) [inline d2l-save-status.js:56]: A nice touch!

File: .github/workflows/release.yml (hunk 3 above)
Comment (devpow112) [reply .github/workflows/release.yml:5]: I'll just change it to add `DEFAULT_BRANCH: master` for now. Don't want to get too crazy with changes.

Comment (dlockhart) [reply .github/workflows/release.yml:5]: This still needs to have `master` here as well.

Comment (devpow112) [reply .github/workflows/release.yml:5]: Oh damn. Yup. copy paste error 😞.

**Review by dlockhart — APPROVED** [review]

**Pull request merged**